packages/react/src/atomic/Box/CHANGELOG.md

```
@@ -2,6 +2,14 @@
 
 A low-level utility component that accepts styled system props to enable custom theme-aware styling
 
+## 2025-11-11 `4.3.2`
+
+#### 🎉 New features
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)